plugins/net_plugin/net_plugin.cpp

```diff
@@ -1195,6 +1206,27 @@ namespace eosio {
         last_handshake_recv(),
         last_handshake_sent()
    {
+      std::istringstream in(limit_str);
+      fc_dlog( logger, "parsing connection endpoint with locale ${l}", ("l", std::locale("").name()));
+      in.imbue(std::locale(""));
+      double limit{0};
+      in >> limit;
+      if( limit > 0.0f ) {
+         std::string units;
+         in >> units;
+         std::regex units_regex{"([KMGT]?[i]?)B/s"};
```

Comment:
I think we should avoid using regex. This seems like a rather simple format to parse via just `std::string`. Also it is not clear to me what the locale is doing here. Seems like we should simply support [K,M,G,T]B/s.

Reply:
Locale-aware parsing means you can use thousands separators in a long number and it will work.  It also means regional decimal separators are supported.  `0.5GB/s` in `en-GB.UTF8`, `0,5GB/s` in `de-AT.UTF8`, and `500'000B/s` in `de-CH.UTF8` will all parse as 500000 bytes per second.

Reply:
Can you add these examples to the PR description.

Reply:
Added.